src/test/resources/pages/studentFeedbackSubmitPageFullyFilled.html

```diff
@@ -375,7 +375,7 @@
                         <textarea class="form-control" cols="100%" name="responsetext-4-0" rows="4">
                            Feedback to team 2.
                         </textarea>
-                        <input name="responseid-4-0" type="hidden" value="${question.id}%SFSubmitUiT.alice.b@gmail.tmt%Team 2"/>
+                        <input name="responseid-4-0" type="hidden" value="${question.id}%Team >'&quot;< 1%Team 2"/>
```

Comment:
The student's data is all sanitized in the db due to #2801, I unsanitized them for the response id. It shows up here this way because, if I recall correctly, " is converted to &quot; during the html comparison process for parsing.